Sources/Dependencies/Internal/Deprecations.swift

```diff
@@ -105,13 +105,14 @@ extension AsyncThrowingStream where Failure == Error {
 
 // MARK: -
 
+@available(*, deprecated)
```

Comment:
A few unrelated changes to get rid of some deprecation warnings in the project/tests.